apps/mymedicare_cb/views.py

```diff
@@ -159,11 +159,24 @@ def mymedicare_login(request, version=1):
     request.session[relay_param_name] = state
     mymedicare_login_url = "%s&%s=%s&redirect_uri=%s" % (
         mymedicare_login_url, relay_param_name, state, redirect)
+    # Check if language was saved server-side for this session
+    language = request.session.get('language', None)
```

Comment:
Naming change here also.

```suggestion
    language = request.session.get('auth_language', None)
```